packages/server/lib/routes.ts

```diff
@@ -104,6 +104,16 @@ export const createCommonRoutes = ({
     next()
   })
 
+  if (!getCtx().coreData.studio || getCtx().coreData.studio?.status === 'IN_ERROR') {
```

Comment:
Beef up the comments

Reply:
Beefed up: [`634109a` (#31033)](https://github.com/cypress-io/cypress/pull/31033/commits/634109a70223aa69af77b1fa5415ce3891556128)